doc/devel/MEP/MEP14.rst

```diff
@@ -115,7 +115,7 @@ requires the installation of TeX-specific font packages, for example,
 and can not use TrueType fonts directly.  Unfortunately, despite the
 different semantics for font selection, the same set of font
 properties are used for each.  This is true of both the
-`FontProperties` class and the font-related `rcParams` (which
+`.FontProperties` class and the font-related `rcParams` (which
```

Comment:
`rcParams` Does not resolve.

Reply:
Actually I just realized that
```
`.rcParams`
```
does resolve :)